docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3.7'
```

Comment:
Since we already have two `Dockerfile`s for running devdocs in production, I would not expect `docker-compose.yml` to be meant for development only.